javascript/javascript2/week3/funVar.js

```diff
@@ -0,0 +1,18 @@
+const arr=[];
```

Comment:
Yeah, this works. Good solution :)

You could also do it like:

const arr=[
function(){console.log("function1 is called")},
function(){console.log("function2 is called")},
function(){console.log("function3 is called")},];